service/src/main/java/com/commercetools/pspadapter/payone/domain/payone/model/wallet/WalletPreauthorizationRequest.java

```diff
@@ -13,10 +13,13 @@
 
     private String wallettype;
 
-    public WalletPreauthorizationRequest(final PayoneConfig config, final ClearingType clearingType) {
+    private int noShipping;
```

Comment:
Why do we need this variable at all? Since mappingUtil is used for all payment methods why we can not just check there if method is paypal and then based on the address data set the noShipping info.

Reply:
Sorry this comment is bit confusing for me.

In Java, we cannot set value to a variable without declaring it. 
To achieve as you said in this comment, 
- We have to declare the 'noShipping'  variable either in **Authorization** request object(which is common request object for all payment types). which is not good because this field is then available for all payment methods.
- So, as you rightly pointed out before, I have created this variable separately only for Wallet Payment method.